Extensions/DeviceVibration/devicevibrationtools.js

```diff
@@ -11,8 +11,9 @@ gdjs.deviceVibration = {};
  * Vibrate the mobile device.
  * @param {number} duration Value in milliseconds.
  */
-gdjs.deviceVibration.vibrationStart = function(duration) {
-  navigator.vibrate([duration]);
+gdjs.deviceVibration.startVibration = function(duration) {
+  if (typeof navigator == "undefined" || !navigator.vibrate) return
+    navigator.vibrate([duration]);
```

Comment:
The way this is indented looks like `navigator.vibrate([duration])` is called if expression inside the if is true, while it's not the case. Could you correct the indentation and add an extra blank line between the if and the navigator.vibrate? Thanks?